File: app/src/main/java/br/com/levimendesestudos/spidermagazine/mvp/contracts/DetalhesMVP.java
package br.com.levimendesestudos.spidermagazine.mvp.contracts;

/**
 * Created by 809778 on 21/12/2016.
 */

public interface DetalhesMVP {

    abstract class Presenter extends BasicPresenter {

    }

    interface View extends BasicView {

        void chamarTelaCapa();

        void carregarDados();
    }
}

File: app/src/main/java/br/com/levimendesestudos/spidermagazine/mvp/contracts/DetalhesMVP.kt
package br.com.levimendesestudos.spidermagazine.mvp.contracts

/**
 * Created by 809778 on 21/12/2016.
 */

interface DetalhesMVP {

    interface Presenter : BasicPresenter

    interface View : BasicView {

        fun configToolbar()

        fun chamarTelaCapa()

        fun carregarDados()
    }
}
